Dockerfile

```diff
@@ -1,24 +1,43 @@
 FROM node:lts-alpine
-ENV NPM_CONFIG_LOGLEVEL warn
 
-# Install "git"
-RUN apk update \
- && apk add alpine-sdk
+ARG VERSION_INFO=noVersion
+
+ENV NPM_CONFIG_LOGLEVEL warn
+ENV PORT=1337
 
 # Set working dir as /app
 WORKDIR /app
 
-# Add sources to /app
-COPY . .
-RUN adduser -S dr4ftuser
-RUN chown dr4ftuser -R .
-USER dr4ftuser
+COPY --chown=node package.json .
+COPY --chown=node package-lock.json .
 
 # Install the dependencies
-RUN npm ci
+RUN npm ci --ignore-scripts
+
+# Update card database
+COPY --chown=node backend/core backend/core
+COPY --chown=node config/ config/
+COPY --chown=node scripts/ scripts/
+RUN npm run download_allsets \
+ && npm run download_booster_rules \
+ && chown node -R data/
```

Comment:
Including this `chown` in the same `RUN` instruction does not increase the size of the image like it would if it were in its own separate `RUN` instruction.